docs/reference/commandline/service_create.md

```diff
@@ -546,6 +546,7 @@ x3ti0erg11rjpg64m75kej2mz-hosttempl
 ## Related information
 
 * [service inspect](service_inspect.md)
+* [service logs](service_logs.md)
 * [service ls](service_ls.md)
 * [service rm](service_rm.md)
 * [service scale](service_scale.md)
```

Comment:
@allencloud why do you want it removed? That rule was added so that the options in the table are not wrapped;

With that rule:

<img width="732" alt="screen shot 2016-12-14 at 14 32 31" src="https://cloud.githubusercontent.com/assets/1804568/21183867/3acaf5c8-c20a-11e6-9a2c-c8eb647fe56f.png">

Without:

<img width="725" alt="screen shot 2016-12-14 at 14 32 17" src="https://cloud.githubusercontent.com/assets/1804568/21183873/42e400ce-c20a-11e6-9d9d-23e53d96daa6.png">